src/Input/Mouse.php

```diff
@@ -45,16 +45,16 @@ public function __construct(Page $page)
     }
 
     /**
-     * @param int        $x
-     * @param int        $y
+     * @param float      $x
+     * @param float      $y
      * @param array|null $options
      *
      * @throws \HeadlessChromium\Exception\CommunicationException
      * @throws \HeadlessChromium\Exception\NoResponseAvailable
      *
      * @return $this
      */
-    public function move(int $x, int $y, ?array $options = null)
+    public function move(float $x, float $y, ?array $options = null)
```

Comment:
This is a breaking change, when the caller has strict types enabled.

Reply:
Why is `getMaximumDistance` a breaking change if it's a private method? And `getPosition` returns an array with no defined types, not even in the docblock.

Reply:
@enricodias 
> Why is `getMaximumDistance` a breaking change if it's a private method?

sorry you're right, getMaximumDistance is not a breaking change, my bad.

>And getPosition returns an array with no defined types, not even in the docblock.

that's kinda a api-break because code that expect `$page->mouse()->getPosition()['y']` to be int may be disappointed. (it broke tests/MouseApiTest.php line 85 test case)

IMO it's a minor API break tho. ["x"=>int,"y"=>int] becomes ["x"=>float,"y"=>float].